skyportal/handlers/api/source.py

```diff
@@ -2740,12 +2753,19 @@ class Validator(Schema):
         alias = self.get_query_argument('alias', None)
         origin = self.get_query_argument('origin', None)
         has_tns_name = self.get_query_argument('hasTNSname', None)
+        has_no_tns_name = self.get_query_argument('hasNoTNSname', None)
         has_been_labelled = self.get_query_argument('hasBeenLabelled', False)
         has_not_been_labelled = self.get_query_argument('hasNotBeenLabelled', False)
         current_user_labeller = self.get_query_argument('currentUserLabeller', False)
         total_matches = self.get_query_argument('totalMatches', None)
         is_token_request = isinstance(self.current_user, Token)
 
+        use_experimental = self.get_query_argument('useExperimental', False)
```

Comment:
Maybe this name could be a little more explicit?

Reply:
Yes. This is just for testing for a few days, I don't intend on anyone using that flag other than me while I benchmark, but you're right it won't hurt to change that name